internal/app/machined/pkg/runtime/v1alpha1/platform/vmware/testdata/metadata-v2.yaml

```diff
@@ -0,0 +1,16 @@
+instance-id: "vmware-test-controlplane-zhnhr"
+local-hostname: "vmware-test-controlplane-zhnhr"
+wait-on-network:
+  ipv4: false
+  ipv6: false
+network:
+  version: 2
+  ethernets:
+    id0:
+      match:
+        macaddress: "00:50:56:b1:81:72"
```

Comment:
we could also match on MAC address, Equinix Metal should have an example of doing so